client/utils/auth.py

```diff
@@ -128,7 +121,7 @@ def do_GET(self):
             self.send_response(200)
             self.send_header("Content-type", "text/html")
             self.end_headers()
-            self.wfile.write(bytes(SUCCESS_HTML, "utf-8"))
+            self.wfile.write(bytes(success_page(SERVER, calnet_id), "utf-8"))
```

Comment:
Similar to Sumukh's comment: instead of prompting for calnet ID on line 92, we should instead prompt for an entire email address (see #80 ):

```
Please enter your Bearfacts email (@berkeley.edu)
Email: example.id@berkeley.edu
```

That way, we hint that students should use their berkeley email, but if they don't have when, they can still enter a gmail address. You can then use that email address on this line.